app/lib/methods/helpers/normalizeMessage.ts

```diff
@@ -9,14 +9,17 @@ function normalizeAttachments(msg: TMsg) {
 	if (typeof msg.attachments !== typeof [] || !msg.attachments || !msg.attachments.length) {
 		msg.attachments = [];
 	}
-	msg.attachments = msg.attachments.map(att => {
-		att.fields = att.fields || [];
-		if (att.ts) {
-			att.ts = moment(att.ts).toDate();
-		}
-		att = normalizeAttachments(att as TMsg);
-		return att;
-	});
+
+	msg.attachments = msg.attachments
+		.filter(att => !!att)
```

Comment:
```suggestion
		.filter(att => att)
```
This is not necessary because this error only occurs when the field is null or undefined.

Reply:
Correct, but we don't know where or since when this error started to occur, so we can't update the types of `msg.attachments` to (IAttachment | null | undefined)[], then, as we don't have this difference, in my opinion, this double-check `!!` shows quick during the read that the att could be different from object IAttachment.

Reply:
As we know, this error could be reproduced by pinning a message with an attachment.
Just FYI :) Probably we'll find a way to fix it a bit later on server side.